db/models.js

```diff
@@ -0,0 +1,14 @@
+const db = require("./connection");
+
+exports.fetchCategories = () => {
+    const sqlQuery = `SELECT * FROM categories;`
+    return db
+    .query(sqlQuery)
+    .then((result) => {
+        return result.rows;
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
Not a particularly useful catch block, great when debugging but for production lets get rid of this. You will however need a catch block within your controller for handling 500 errors.